### ISSUE TITLE
`SqlNoDataSourceInspection` is not recognised as a valid inspection

### ISSUE THREAD

**Issue by zerkms** [kalessil/phpinspectionsea#1268]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended), v3.0.12.2  |
| Language level | PHP 7.2 (irrelevant)                        |

Current behaviour
```php
<?php

/** @var PDO $db */
$db = null;

/** @noinspection SqlNoDataSourceInspection */
$stmt = $db->prepare('SELECT foo FROM tbl');
```

For this code `SqlNoDataSourceInspection` inspection is marked as "Unknown inspection".

I originally reported it at https://youtrack.jetbrains.com/issue/WI-46021 but then found that it comes from this plugin not the IDE.

--

Expected behaviour
It's a valid inspection that comes with phpstorm

--

Environment details
PhpStorm 2019.1
Build #PS-191.6183.95, built on March 28, 2019
Licensed to Ivan Kurnosov
Subscription is active until September 5, 2019
JRE: 1.8.0_202-release-1483-b39 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-46-generic

---

**Comment by kalessil** [comment]
Fixed (331b743789b09465c333903d599ae0819d03b263)!
